RecoEgamma/EgammaIsolationAlgos/plugins/isolation_cones/PhotonPFIsolationWithMapBasedVeto.cc

```diff
@@ -132,9 +132,9 @@ bool PhotonPFIsolationWithMapBasedVeto::isInIsolationCone(const reco::CandidateP
   // dealing here with patObjects: miniAOD case
   if ( aspacked.get() )    
   {
-    inFootprint = isInFootprint(aspat_photonptr ->associatedPackedPFCandidates(),aspacked);
-    
-    //checking if the charged candidates come from the appropriate vertex
+    //inFootprint = isInFootprint(aspat_photonptr ->associatedPackedPFCandidates(),aspacked);
```

Comment:
Dereference of aspat_photonptr will work only if the input photon is a pat::Photon.
You configured the code to read reducedEgamma photons, which is made of reco::Photon.

This code will need to be updated to handle non-pat photons.